public/components/datasources/components/manage/data_connection.tsx

```diff
@@ -103,6 +107,48 @@ export const DataConnection = (props: any) => {
     },
   ];
 
+  const QueryOrAccelerateData = () => {
+    console.log(chrome);
+    switch (datasourceDetails.connector) {
+      case 'S3GLUE':
+        return (
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <EuiCard
+                icon={<EuiIcon size="xxl" type="discoverApp" />}
+                title={'Query data'}
+                description="Query your data in Data Explorer or Observability Logs."
+                onClick={() => application!.navigateToApp(observabilityLogsID)}
```

Comment:
Need another set of eyes here that this is the right way to navigate between apps

Reply:
IMO This is good!